hstool/models.py

```diff
@@ -52,6 +52,7 @@ def south_field_triple(self):
 
 
 class Figure(Model):
+    draft = BooleanField(default=True)
```

Comment:
We should keep this, is the same approach as we have in `flis.flip`. Usually, a status field with states like: New, Draft, Published, Archived will cover more use cases, but it's better to KISS and DRY.